src/tests/blueprint/t_blueprint_mpi_relay.cpp

```diff
@@ -113,28 +116,185 @@ TEST(blueprint_mpi_relay, basic_use)
+{
+    Node io_protos;
+    relay::io::about(io_protos["io"]);
+    bool hdf5_enabled = io_protos["io/protocols/hdf5"].as_string() == "enabled";
+
+    // only run this test if hdf5 is enabled
+    if(!hdf5_enabled)
+    {
+        CONDUIT_INFO("hdf5 is disabled, skipping hdf5 dependent test");
+        return;
+    }
+
+    MPI_Comm comm = MPI_COMM_WORLD;
+    int par_rank = mpi::rank(comm);
+    int par_size = mpi::size(comm);
+
+    Node dset, v_info;
+    blueprint::mpi::mesh::examples::spiral_round_robin(5,
+                                                       dset,
+                                                       comm);
+
+    // check verify
+    EXPECT_TRUE(blueprint::mpi::mesh::verify(dset,v_info,comm));
+
+    // locally, expect 3 domas for rank 0, 2 for rank 1
+    if(par_rank == 0)
+        EXPECT_EQ(blueprint::mesh::number_of_domains(dset),3);
+    else
+        EXPECT_EQ(blueprint::mesh::number_of_domains(dset),2);
+
+    // globally, expect par_size domains
+    EXPECT_EQ(blueprint::mpi::mesh::number_of_domains(dset,comm),5);
+
+    string protocol = "hdf5";
+    string output_base = "tout_blueprint_mpi_relay_spiral_mpi_dist_5doms";
+    // what we want:
+    // relay::mpi::io::blueprint::save_mesh(dset, output_path,"hdf5");
+    conduit::relay::mpi::io::blueprint::save_mesh(dset,
+                                                  output_base,
+                                                  "hdf5",
+                                                  comm);
+    MPI_Barrier(comm);
+
+    // read this back using load_mesh, should diff clean
+    string output_root = output_base + ".cycle_000000.root";
+    Node n_read, n_diff_info;
+    conduit::relay::mpi::io::blueprint::load_mesh(output_root,
+                                                  n_read,
+                                                  comm);
+}
+
+//-----------------------------------------------------------------------------
+TEST(blueprint_mpi_relay, mpi_mesh_examples_spiral_1dom)
+{
+    Node io_protos;
+    relay::io::about(io_protos["io"]);
+    bool hdf5_enabled = io_protos["io/protocols/hdf5"].as_string() == "enabled";
+
+    // only run this test if hdf5 is enabled
+    if(!hdf5_enabled)
+    {
+        CONDUIT_INFO("hdf5 is disabled, skipping hdf5 dependent test");
+        return;
+    }
+
+    MPI_Comm comm = MPI_COMM_WORLD;
+    int par_rank = mpi::rank(comm);
+    int par_size = mpi::size(comm);
+
+    Node dset, v_info;
+    blueprint::mpi::mesh::examples::spiral_round_robin(1,
+                                                       dset,
+                                                       comm);
+
+    // check verify
+    EXPECT_TRUE(blueprint::mpi::mesh::verify(dset,v_info,comm));
+
+    // locally, expect 3 domas for rank 0, 2 for rank 1
+    if(par_rank == 0)
+        EXPECT_EQ(blueprint::mesh::number_of_domains(dset),1);
+    else
+        EXPECT_EQ(blueprint::mesh::number_of_domains(dset),0);
```

Comment:
Haven't required braces for code blocks historically been a part of Conduit style? This seems to be the case looking briefly at [`conduit_node.cpp`](https://github.com/LLNL/conduit/blob/master/src/libs/conduit/conduit_node.cpp#L408), but I'm not certain this style is ubiquitous across the code.

Reply:
good catch, I'l resolve